apollo-rxsupport/build.gradle

```diff
@@ -0,0 +1,61 @@
+buildscript {
+  dependencies {
+    classpath dep.androidPlugin
+  }
+}
+
+apply plugin: 'com.android.library'
+
+android {
+  compileSdkVersion androidConfig.compileSdkVersion
+  buildToolsVersion androidConfig.buildToolsVersion
+
+  compileOptions {
+    sourceCompatibility JavaVersion.VERSION_1_7
+    targetCompatibility JavaVersion.VERSION_1_7
+  }
+
+  lintOptions {
+    textReport true
+    textOutput 'stdout'
+    ignore 'InvalidPackage'
+  }
+
+  dexOptions {
+    preDexLibraries = isCi
+  }
+
+  defaultConfig {
+    minSdkVersion 11
+    testInstrumentationRunner 'android.support.test.runner.AndroidJUnitRunner'
+  }
+}
+
+dependencies {
+  compile dep.jsr305
+  compile dep.supportAnnotations
+  compile dep.okHttp
+  compile dep.moshi
+  compile dep.rxjava
+  compile project(":apollo-runtime")
+  compile project(":apollo-api")
```

Comment:
both of those should be provided because rx-support should assume those would be available at runtime


(Also, `compile project(":apollo-runtime")` doesn't bring the transitive deps with it. That's why when having `compile project(":apollo-api")`  under runtime, still required you to bring that in.